test/pylib/scylla_cluster.py

```diff
@@ -127,7 +127,8 @@ def make_scylla_conf(workdir: pathlib.Path, host_addr: str, seed_addrs: List[str
     '--abort-on-lsa-bad-alloc', '1',
     '--abort-on-seastar-bad-alloc',
     '--abort-on-internal-error', '1',
-    '--abort-on-ebadf', '1'
+    '--abort-on-ebadf', '1',
+    '--logger-log-level', 'raft_topology=debug',
```

Comment:
Shouldn't we do it for topology suits only?

Reply:
Doesn't hurt to do it for others as well.